Doc/library/configparser.rst

```diff
@@ -942,7 +942,13 @@ interpolation if an option used is not defined elsewhere. ::
 ConfigParser Objects
 --------------------
 
-.. class:: ConfigParser(defaults=None, dict_type=dict, allow_no_value=False, delimiters=('=', ':'), comment_prefixes=('#', ';'), inline_comment_prefixes=None, strict=True, empty_lines_in_values=True, default_section=configparser.DEFAULTSECT, interpolation=BasicInterpolation(), converters={})
+.. class:: ConfigParser(defaults=None, dict_type=dict, allow_no_value=False, \
+                        delimiters=('=', ':'), comment_prefixes=('#', ';'), \
```

Comment:
I think there should be an asterisk * item, because _allow_unnamed_section_, and all other parameters from _delimiters_, are keyword-only.
```suggestion
.. class:: ConfigParser(defaults=None, dict_type=dict, allow_no_value=False, *, \
                        delimiters=('=', ':'), comment_prefixes=('#', ';'), \
```